addons/website_event/models/event.py

```diff
@@ -2,12 +2,11 @@
 
 import re
 
-from openerp import models, fields, api, _
-from openerp import SUPERUSER_ID
+from openerp import api, fields, models, _
 from openerp.addons.website.models.website import slug
 
 
-class event(models.Model):
+class Event(models.Model):
```

Comment:
At end of file missed change event ==> Event  (super(event, self)._track_subtype)